### DISCUSSION TITLE
Direction of items in pmenu

### DISCUSSION THREAD

**Discussion by GordianDziwis** [hrsh7th/nvim-cmp#663] [Q&A]
On default the fist item in the popup menu is always the top one.
But I would like to configure it in a way that the best item is always closest to the insert location. So if the pmenu opens above the place where I am typing the best item is the bottom item in the pmenu. Does anyone know howto do this?

**Comment by hrsh7th** [comment]
Currently, no.

Hm...what is the correct behaivior of `<C-n>` in tjis situation?

**Reply by GordianDziwis** [reply]
Telescope has 'move_selection_down' and 'move_selection_worse' for handling this case. https://github.com/nvim-telescope/telescope.nvim/blob/5f37fbfa837dfee7ecd30f388b271f4a71c0a9e0/doc/telescope.txt#L1866

**Comment by Shougo** [comment]
Hm.  But the default completion menu has not the feature.

**Reply by GordianDziwis** [reply]
Then it is best to start with an issue for neovim or vim?

**Reply by Shougo** [reply]
nvim-cmp recommends the original completion menu, it is not related Vim/neovim completion menu though.

**Reply by GordianDziwis** [reply]
https://github.com/vim/vim/issues/9399
